.golangci.yml

```diff
@@ -104,7 +104,6 @@ linters-settings:
     include-go-root: true
     packages-with-error-message:
       # See https://github.com/open-telemetry/opentelemetry-collector/issues/5200 for rationale
-      - sync/atomic: "Use go.uber.org/atomic instead of sync/atomic"
```

Comment:
I think we should disallow go.uber.org/atomic now.

Reply:
Done